extensions/bodyparser-msgpack/package.json

```
@@ -0,0 +1,53 @@
+{
+  "name": "@loopback/bodyparser-msgpack",
+  "version": "0.0.1",
+  "description": "@loopback/bodyparser-msgpack",
+  "keywords": [
+    "loopback-extension",
+    "loopback"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "engines": {
+    "node": ">=10.16"
+  },
+  "scripts": {
+    "build": "lb-tsc",
+    "build:watch": "lb-tsc --watch",
+    "pretest": "npm run clean && npm run build",
+    "test": "lb-mocha \"dist/__tests__/**/*.js\"",
+    "clean": "lb-clean dist *.tsbuildinfo .eslintcache"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/strongloop/loopback-next.git",
+    "directory": "extensions/bodyparser-msgpack"
```

I think we may be adding more bodyparsers in the future, perhaps it's time to introduce more directory structure?

```suggestion
    "directory": "extensions/bodyparsers/msgpack"
```